docs/resources/samples/email-chart-image.md

```diff
@@ -0,0 +1,91 @@
+---
+title: 'Email chart image'
+description: 'Learn how to use Office Scripts and Power Automate to extract and email chart image.'
+ms.date: 03/04/2021
+localization_priority: Normal
+---
+
+# Use Office Scripts and Power Automate to email chart and table images
+
+Shows how to use Office Scripts and Power Automate actions to recalculate, create a chart, and extract images as base-64 strings to be used later to send by email.
```

Comment:
There are a lot of implementation details here that could be omitted in favor of highlighting the scenario.
```suggestion
This sample uses Office Scripts and Power Automate actions to create a chart and send that chart as an image by email.
```